packages/chains/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@wagmi/chains",
-  "description": "A collection of chains for wagmi",
+  "name": "@wagmi-references/chains",
+  "description": "A collection of chains for wagmi (with ew chains)",
```

Comment:
@nichonien The first part of the package name (prefixed with the `@`, until the `/`) is the scope of the name. https://docs.npmjs.com/cli/v9/using-npm/scope
So for packages published by EW, we should use the `@energyweb` scope (as we used for ew-credentials packages https://github.com/energywebfoundation/ew-credentials#description , for example), in my opinion. Perhaps a package name like `@energyweb/wagmi-chains` makes sense.
How do you see this?

Reply:
Hmm, I agree with you. `@energyweb/wagmi-chains` makes more sense.